codegen/java-gen/src/test/resources/tests/action-service/named-new-style/generated-managed/org/example/service/MyServiceNamedActionRouter.java

```diff
@@ -3,7 +3,7 @@
 import akka.NotUsed;
 import akka.stream.javadsl.Source;
 import com.google.protobuf.Empty;
-import kalix.javasdk.action.Action.Effect;
+import kalix.javasdk.action.AbstractAction.Effect;
```

Comment:
I just realised that we move Action to AbstractAction because the Effect changes its location and that can potentially break existing code. 

@franciscolopezsancho / @aludwiko

Reply:
Wasn't that the same with workflows, and we did it anyway? Or maybe we just haven't spotted it?

Reply:
Well spotted. Nice. 
Are there any other breaking changes we are aware we would like to do?
How about we group them all together and release 2.0? In a few weeks I mean.

Meanwhile we could release the original PR or just wait until releasing this breaking change, if we decide to do it, ofc.
WDYT guys?